### PR TITLE
Feature/highlight search results

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

- removed code the generated snippets with start/end points
- removed bold (`<strong>`) and reverted to default highlighting (`<em>`) in elasticsearch
- updated swagger spec to reflect that MatchDetails no longer exists

### How to review

Check that tests pass, relevant code has been removed and swagger spec is correct. 

### Who can review

Anyone but me.

### CODE REVIEW

File: transformer/testdata/search_example.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
                         },
                         "highlight": {
                             "description.title": [
-                                "<strong>Housing</strong>"
+                                "<em>Housing</em>"
```
Comment (armstrongb) [inline transformer/testdata/search_example.json:82]: `<em class="highlight">` is the class used in the design system (https://ons-design-system.netlify.app/patterns/see-important-information/)

**Review by armstrongb — CHANGES_REQUESTED** [review]
To match the design system highlighting.

**Review by bpathak-ons — COMMENTED** [review]
Appropriate unit tests, with more not-null values, should be added/updated in view of significant changes in slice algorithm in code. Pls don't merge without adding more unit tests and if possible, with regression testing for this branch.

**Review by bpathak-ons — APPROVED** [review]
looks ok to me

**Review by armstrongb — APPROVED** [review]
LGTM - from an examination of the code it looks like highlight class has been added where relevant.

**Pull request merged**